### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by cllns**
Please see my proposed changes for the README. I wanted to provide more info about what exactly this repo is, and then I streamlined some other content.

Let me know what you think, happy to tweak even tiny things (or abandon some of the changes altogether, like 🌸, even though I love it so).

You can preview these rendered changes here: https://github.com/cllns/hanami/tree/update-readme

### CODE REVIEW

**Comment by oana-sipos** [comment]
Thumbs up for your changes @cllns. Looks neat and has a good flow of info 👍

File: README.md [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
+[![Coverage](https://coveralls.io/repos/hanami/hanami/badge.svg?branch=master)](https://coveralls.io/r/hanami/hanami)
+[![Code Climate](https://codeclimate.com/github/hanami/hanami.svg)](https://codeclimate.com/github/hanami/hanami)
+[![Dependencies](https://gemnasium.com/hanami/hanami.svg)](https://gemnasium.com/hanami/hanami)
+[![Inline docs](http://inch-ci.org/github/hanami/hanami.svg)](http://inch-ci.org/github/hanami/hanami)
+
+# Hanami :cherry_blossom:
 
 The web, with simplicity.
 
 ## Frameworks
```
Comment (davydovanton) [inline README.md]: @cllns hanami is one framework. maybe is better to use the single form here. What do you think?

Comment (cllns) [inline README.md]: I think @jodosha refers to the individual parts as (single-purpose) frameworks. I typically think of a framework as being more general purpose, like `hanami/hanami`, and consider the other parts "libraries". What do you think, Luca?

Comment (cllns) [inline README.md]: Thanks @b264. I thought I remembered the term "micro-libraries" but didn't remember where.

**Comment by cllns** [comment]
Feel free to merge 🙃

File: README.md (hunk 1 above)
Comment (TiteiKo) [inline README.md]: 👍  for libraries instead of frameworks as the title here

File: README.md [hunk 2]
```diff
@@ -1,23 +1,52 @@
...
-* [**Hanami::View**](https://github.com/hanami/view) - Presentation with a separation between views and templates
-* [**Hanami::Controller**](https://github.com/hanami/controller) - Full featured, fast and testable actions for Rack
+* [**Hanami::Utils**](https://github.com/hanami/utils) - Ruby core extensions and class utilities
 
 These components are designed to be used independently or together in a Hanami application.
-If you aren't familiar with them, please take time to go through their READMEs.
+
+## Installation
+__Hanami__ supports Ruby (MRI) 2.3+
+
```
Comment (Neodelf) [inline README.md]: Whether or not to specify here the version jruby?

File: README.md [hunk 3]
```diff
@@ -1,23 +1,52 @@
...
 * [**Hanami::Assets**](https://github.com/hanami/assets) - Assets management for Ruby
-* [**Hanami::View**](https://github.com/hanami/view) - Presentation with a separation between views and templates
-* [**Hanami::Controller**](https://github.com/hanami/controller) - Full featured, fast and testable actions for Rack
+* [**Hanami::Utils**](https://github.com/hanami/utils) - Ruby core extensions and class utilities
 
 These components are designed to be used independently or together in a Hanami application.
-If you aren't familiar with them, please take time to go through their READMEs.
+
+## Installation
+__Hanami__ supports Ruby (MRI) 2.3+
```
Comment (Neodelf) [inline README.md]: Whether or not to specify here the version `jruby`?

File: README.md [hunk 4]
```diff
@@ -1,23 +1,52 @@
...
-Hanami combines small yet powerful frameworks:
+Hanami is a **full-stack** Ruby web framework.
+It's made up of smaller, single-purpose libraries.
 
-* [**Hanami::Utils**](https://github.com/hanami/utils) - Ruby core extensions and class utilities
-* [**Hanami::Router**](https://github.com/hanami/router) - Rack compatible HTTP router for Ruby
+This repository is for the full-stack framework,
+which provides the glue that ties all the parts together.
+
+Here are the smaller single-purpose frameworks:
```
Comment (b264) [inline README.md]: Libraries or frameworks?

Comment (cllns) [inline README.md]: thank you! Actually just got rid of this because we had the same wording a couple lines up

File: README.md (hunk 3 above)
Comment (jodosha) [inline README.md]: @Neodelf We don't support JRuby for `hanami` yet. All the other `hanami-*` libraries already support it.

Comment (Neodelf) [inline README.md]: @jodosha thank you. I'm sorry for my carelessness

**Pull request merged**